lua/ttt2/libraries/spawn.lua

```diff
@@ -27,21 +25,29 @@ local spawnTypes = {
 	"info_player_teamspawn"
 }
 
+-- returns the player size as a vector
+local function GetPlayerSize(ply)
+	local bottom, top = ply:GetHull()
 
+	return -1 * bottom + top
```

Comment:
Does vector math support unary negation? If yes, ```return -bottom + top``` would fit better.

Reply:
or `return top - bottom`

Reply:
I don't like lebroomers suggestion, therefore I used `-1  * x`, but saibotks solution is rather nice. Thanks. Too easy!